gwtlib/display.py

```
@@ -272,56 +255,52 @@ def print_branch(b, kind):
+    fetch_failures = []
+
+    # Get local branches (single fetch, used for both collection and categorization)
+    local_branches = set()
+    try:
+        result = run_git_command(
+            ["for-each-ref", "--format=%(refname:short)", "refs/heads/"], git_dir
+        )
+        for branch in result.stdout.strip().split("\n"):
+            if branch:
+                local_branches.add(branch)
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)